### ISSUE TITLE
2.6.0 (e) - Sponsor's sorted incorrectly

### ISSUE THREAD

**Issue by insaneblaine** [geoffhumphrey/brewcompetitiononlineentry#1440]
**BCOE&M Info**
Version: 2.6.0 (e) 
Installation URL: motownmash.brewcompetition.com
Hosted Installation? Yes 

**Describe the Bug**
Sponsors are being sorted in a nondeterministic order. Previously sponsors were sorted by sponsor level and then alphabetically within each level. Now it's in an order that I cannot figure out.

**To Reproduce**
Create a bunch of sponsors with different alphabetic names and sponsor levels. On sponsor page, they will be sorted using an undiscoverable algorithm

**Expected Behavior**
Sponsors sorted by level then alphabetically within each level.

**Comment by geoffhumphrey** [comment]
Hi @insaneblaine, thanks for reporting. For reasons I cannot recall, the default SQL query was changed when accessing Sponsor data. I've fixed that in your installation since it's hosted and will fix all other hosted installations soon.

For those self-hosting their BCOE&M instance, download the [updated /includes/db/sponsors.db.php file from here](https://github.com/geoffhumphrey/brewcompetitiononlineentry/commit/cc74a359c065a1fc3bbc9d768153abbb4d0411b0) and replace on your server.
